### DISCUSSION TITLE
Téléchargement de données rentrées dans l'outil suivi d'achats

### DISCUSSION THREAD

**Discussion by alemangui** [betagouv/ma-cantine#989] [Retours utilisateur·rice]
Plusieurs utilisateurs.trices ont remonté le besoin de télécharger les données renseignées sous forme de tableur Excel (pour pouvoir les exploiter de leur côté).

**Comment by hfroot** [comment]
Cette idée a été remontée dans mattermost récemment, est-ce qu'on a besoin de plus de détail avant aborder cette fonctionnalité dans un sprint ?
De mon côté il me semble assez clair pour au moins un premier pas.
On peut utiliser ça pour un construire un endpoint assez rapidement https://www.django-rest-framework.org/api-guide/renderers/#xlsx-binary-spreadsheet-endpoints
